### DISCUSSION TITLE
Wildcard certs?

### DISCUSSION THREAD

**Discussion by benlabbeus** [n3wt0n/AzureWebAppSSLManager#32] [Q&A] [answered]
Does this support wildcard certs?

**Comment by n3wt0n** [comment, marked as answer]
Yes, it does.

You can configure it this way:
https://github.com/n3wt0n/AzureWebAppSSLManager/blob/64ef404cd28306886566434a9d5ea2407f20fa8d/SampleJsonConfig/appproperties.json#L3
